update-versions.cjs

```diff
@@ -0,0 +1,17 @@
+const fs = require('fs')
+
+const version = process.argv[2]
+if (!version) {
+  console.error(`usage: ${__filename} VERSION`)
+  process.exit(1)
+}
+
+const lines = fs.readFileSync('dist/itkConfig.js', { encoding: 'utf-8' }).split('\n')
```

Comment:
I can do it for `dist/itkConfig.js` and `dist/itkConfigDevelopment.js`. We don't need to do it for them in `src`, since this is run after a build, when a release is being published.